src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -22,6 +22,8 @@ namespace GitUI
 {
     public sealed partial class FileStatusList : GitModuleControl
     {
+        private const string _showAllDifferencesItemName = "ShowDiffForAllParentsText";
```

Comment:
```suggestion
        private const string ShowAllDifferencesItemName = nameof(ShowDiffForAllParentsText);
```

Reply:
I have followed the naming conventions for `private` constants:

![image](https://github.com/user-attachments/assets/d52656a9-7dac-4de7-a864-b934f984f3de)

![image](https://github.com/user-attachments/assets/fe521729-97b5-49cf-93d3-e1dd2872268c)

---

There is no such identifier `ShowDiffForAllParentsText` except in `TranslatedStrings`.
I have aligned the moved / new identifiers as well.